### DISCUSSION TITLE
Execution of Scorecard on Nexus repository manager

### DISCUSSION THREAD

**Discussion by PiyushArena** [ossf/scorecard#3856] [General]
Can someone confirm that we can execute the scorecard on Nexus repository manager

**Comment by spencerschrock** [comment]
Can you provide more details? You run scorecard against source repositories. Which repository are you trying to analyze?

**Reply by PiyushArena** [reply]
It would be more like running scorecard on maven central or where exact jars or dependencies would be present

**Reply by spencerschrock** [reply]
We don't currently support looking up Maven packages. The related issue is https://github.com/ossf/scorecard/issues/2656
